modules/kg-schemas/src/main/resources/schemas/nexus/core/datadownload/v0.1.0.json

```diff
@@ -0,0 +1,146 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0",
+    "{{base}}/contexts/nexus/core/standards/v0.1.0",
+    {
+      "nxcv": "https://bbp-nexus.epfl.ch/vocabs/nexus/core/v0.0.1/",
+      "sqvsh": "{{base}}/schemas/nexus/schemaorg/quantitativevalue/v0.1.0/shapes/",
+      "this": "{{base}}/schemas/nexus/core/datadownload/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "owl:Ontology",
+  "imports": [
+    "{{base}}/schemas/nexus/schemaorg/quantitativevalue/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:DataDownloadShape",
+      "@type": "sh:NodeShape",
+      "label": "",
+      "targetClass": "nxcv:DataDownload",
+      "property": [
+        {
+          "path": "nxcv:contentSize",
+          "node": "this:ContentSizeQuantitativeValueShape",
+          "maxCount": 1
+        },
+        {
+          "path": "nxcv:digest",
+          "node": "this:DigestShape",
+          "maxCount": 1
+        },
+        {
+          "or": [
+            {
+              "path": "nxcv:downloadURL",
+              "datatype": "xsd:anyURI",
+              "maxCount": 1
+            },
+            {
+              "path": "nxcv:accessURL",
+              "datatype": "xsd:anyURI",
+              "maxCount": 1
+            }
+          ]
+        },
+        {
+          "path": "nxcv:mediaType",
+          "datatype": "xsd:string",
+          "minCount": 1,
+          "maxCount": 1
+        },
+        {
+          "path": "nxcv:storage",
+          "name": "Storage",
+          "description": "Storage",
+          "node": "this:StorageShape",
+          "maxCount": 1
```

Comment:
should we always require a storage?

Reply:
I guess I missed part of this conversation, but it's not clear what Storage is.  It's not defined here:
https://www.w3.org/TR/vocab-dcat/
or here:
http://schema.org/DataDownload

Reply:
The storage is indeed not part of schema.org/DataDownload and w3c dcat specs . I was not able to find a spec that introduces it but I think we need it:
to record the type of storage: web, archive,…
to record what storage service is used: zenodo, …. allowing data consumers to target data they know they can handle because they have the right client (auth, ..)

I'll keep doing research on it to find if there are specs that support storage definition.